src/utils/uniswap.js

```diff
@@ -24,28 +24,23 @@ import {
 } from '@uniswap/sdk';
 import { toChecksumAddress } from '@netgum/utils';
 import { BigNumber } from 'bignumber.js';
+import { getEnv } from 'configs/envConfig';
 
 import ROUTER_ABI from 'abi/uniswapRouter.json';
-import { isProdEnv } from 'utils/environment';
 import { encodeContractMethod } from 'services/assets';
 import { ETH } from 'constants/assetsConstants';
 import { ALLOWED_SLIPPAGE } from 'constants/exchangeConstants';
 
 import type { Asset } from 'models/Asset';
 
-const isMainnet = isProdEnv;
-export const chainId = isMainnet ? ChainId.MAINNET : ChainId.KOVAN;
+type UniswapChainId = ChainId.MAINNET | ChainId.KOVAN;
+
+export const getChainId = (): UniswapChainId => {
```

Comment:
```suggestion
export const getUniswapChainId = (): UniswapChainId => {
```
`getChainId` seems overly general name for uniswap-related function.